cypress/e2e/0-JoinBet/footer/footer.cy.js

```diff
@@ -10,47 +11,66 @@ describe('Localization. Casino. Footer.', () => {
             bonusModalClose();
             Object.values(footer.links).forEach((selector, index) => {
                 cy.get(selector).should('have.text', Object.values(values)[index])
-            })
+            });
             cy.get(footer.titles)
                 .should('have.length', 5)
                 .each((element, index) => {
                     cy.get(element).should('have.text', Object.values(values.headLines)[index])
-                })
-        })
-    })
-})
+                });
+        });
+    });
+});
 
 describe('Casino. Footer.',()=> {
     beforeEach(() => {
         cy.authenticateUrl('https://st2-gdjsmns.hk/ru/casino/lobby');
         bonusModalClose();
-    })
+    });
     it('Переход по ссылкам', () => {
         Object.values(footer.links).forEach((selector, index) => {
-            cy.scrollTo('bottom')
+            cy.scrollTo('bottom');
             cy.get(selector)
-                .click({force: true})
-            cy.url().should('include', urls[index])
-            cy.status200(urls[index])
-            cy.go('back')
-        })
+                .click({force: true});
+            cy.url().should('include', urls[index]);
+            cy.status200(urls[index]);
+            cy.go('back');
+        });
     });
     it('Клик по логотипу', () => {
-        cy.scrollTo('bottom')
+        cy.scrollTo('bottom');
         cy.get(footer.logo)
-            .click({force: true})
-        cy.url().should('include', urls[0])
-        cy.status200(urls[0])
+            .click({force: true});
+        cy.url().should('include', urls[0]);
+        cy.status200(urls[0]);
     });
     it('Отображение платёжных систем', () => {
-        cy.scrollTo('bottom')
+        cy.scrollTo('bottom');
         Object.values(footer.payment).forEach((selector) => {
             cy.get(selector)
                 .should('be.visible')
                 .and('have.attr', 'src')
                 .then((element) => {
                     cy.status200(element)
-                })
-        })
+                });
+        });
+    });
+});
+
+describe('Registration. Casino. Header.',() => {
+    it('Проверка списка валют', () => {
```

Comment:
```suggestion
    it(`Проверить список валют в дропдауне "Валюта"` , () => {
```
Название взять из Тест-рейл для всех авто-тестов

Reply:
Сделано